gtest/src/manager.rs

```diff
@@ -446,6 +450,61 @@ impl ExtManager {
 
         core_processor::handle_journal(journal, self);
     }
+
+    #[allow(dead_code)]
```

Comment:
I prefer allowing `unused` for all such places, due to logic use cases. Some minimal docs would be also good.